js/main.js

```diff
@@ -5,18 +5,69 @@ function getRandomInteger (min, max) {
     max = newMax;
     min = newMin;
   }
-
   if (max < 0 || min < 0) {
     throw new Error('Аргументы должны быть не меньше нуля');
   }
-
   return Math.round(min + Math.random() * (max - min));
 }
 
-getRandomInteger(0, 10);
-
 function getStrLength (str, length) {
   return str.length < length;
 }
 
 getStrLength ('Привет', 10);
+
+const getRandomArrayElement = (array) => array[getRandomInteger(0, array.length - 1)];
+
+function createIdGenerator () {
+  let lastGeneratedId = 0;
+  return function () {
+    lastGeneratedId += 1;
+    return lastGeneratedId;
+  };
+}
```

Comment:
```suggestion
 let counter = 0;
  
function count () {
    lastGeneratedId += 1;
  };
}
```

на каждый вызов этой функции будет увеличиваться каунтер. Не стоит усложнять код.